includes/class-wc-gateway-cardknox-applepay.php

```diff
@@ -27,7 +36,7 @@ public function __construct()
 
         $methodDescription = '<strong class="important-label" style="color: #e22626;">Important: </strong>';
         $methodDescription .= 'Please complete the Apple Pay Domain Registration ';
-        $methodDescription .= '<a target="_blank" href="https://portal.cardknox.com/account-settings/payment-methods">';
+        $methodDescription .= '<a target="_blank" href="https://portal2.solapayments.com/account-settings/payment-methods">';
```

Comment:
Why portal2?

Reply:
Hello @mstein-ck 
As I mentioned in pr #78, our URL has changed, so we have added this URL
(Update required: Replace all occurrences of https://portal.cardknox.com/ with https://portal2.solapayments.com/ to reflect the new portal URL).

Reply:
Can you use https://portal.solapayments.com/

Reply:
> Can you use https://portal.solapayments.com/

Hello @mstein-ck 

We changed the URL from https://portal2.solapayments.com/ to https://portal.solapayments.com/
Please review it.